qt/scientific_interfaces/Indirect/IndirectDataManipulation.ui

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>IndirectDataManipulation</class>
+ <widget class="QMainWindow" name="IndirectDataManipulation">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>926</width>
+    <height>1021</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Indirect Data Reduction</string>
```

Comment:
Should the window title be "Indirect Data Manipulation" to distinguish it from the Indirect Data Reduction UI which already has this title?

Reply:
I have another PR ready for changing a lot of file labels to inelastic. I will change the title for that, rather than just chnaging it now only to rename it imediatly.